clang/www/cxx_dr_status.html

```diff
@@ -12702,7 +12702,7 @@ <h2 id="cxxdr">C++ defect report implementation status</h2>
     <td><a href="https://cplusplus.github.io/CWG/issues/2149.html">2149</a></td>
     <td>drafting</td>
     <td>Brace elision and array length deduction</td>
-    <td align="center">Not resolved</td>
+    <td title="Clang 3.1 implements 2024-04 resolution" align="center">Not Resolved*</td>
```

Comment:
Still not too familiar w/ DRs candidly; is ‘Not Resolved’ intended here?

Reply:
As I mentioned in https://github.com/llvm/llvm-project/pull/90079#issuecomment-2077691304, CWG index page that we fetch from is not updated yet. I checked with CWG chair regarding most up-to-date source of CWG issue status, and I'll prepare an update to `make_cxx_dr_status` script to reduce amount of confusions like this one.

Reply:
> CWG index page that we fetch from

Ah, I see; I didn’t know that that’s how it works. Makes sense.